community/code_of_conduct.md

```diff
@@ -1,43 +1,80 @@
 
-- Using welcoming and inclusive language
-- Being respectful of differing viewpoints and experiences
-- Gracefully accepting constructive criticism
-- Focusing on what is best for the community
-- Showing empathy towards other community members
+* Practice empathy and humility.
+* Assume competence first.
+* Assume that everyone we work with is doing their best work for the community.
+* Listen carefully and actively.
+* Ask questions, and seek to understand your partners' context.
+* Encourage other people to listen as much as they speak.
+* Treat other people's identities and cultures with respect. Make an effort to say people's names correctly and refer to them by their chosen pronouns.
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community.
+* Showing empathy towards other community members.
 
-Examples of unacceptable behavior by participants include:
+## Unacceptable behavior
 
-- The use of sexualized language or imagery and unwelcome sexual attention or advances
-- Trolling, insulting/derogatory comments, and personal or political attacks
-- Public or private harassment
-- Publishing other's private information, such as a physical or electronic address, without explicit permission
-- Other conduct which could reasonably be considered inappropriate in a professional setting
+To help colleagues understand the kinds of behaviors that are illegal or run counter to the culture we seek to foster, we've listed below actions that are counter to our values.  We've also included steps to take if you encounter behavior that runs contrary to this policy.
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Sustained disruption of meetings, talks, or discussions.
+* Mocking someone's real or perceived accent or first language.
+* Repeatedly interrupting or talking over other people in meetings and discussions.
+* Negative or offensive remarks based on gender expression, mental illness, socioeconomic status or background, neuro(a)typicality, physical appearance, or body size.
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
 
 ## Our Responsibilities
 
-Moderators are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+Staff is responsible for clarifying the standards of acceptable
```

Comment:
I think this should link to who staff includes.

Reply:
Agreed